### DISCUSSION TITLE
Playback doesn't work in Plex for Windows (workaround in comments)

### DISCUSSION THREAD

**Discussion by chrisbenincasa** [chrisbenincasa/tunarr#1381] [General]
Haven't found a common set of criteria here yet. We've seen at least 2 reports of this so far. It doesn't seem to work in Plex for Windows desktop, unclear how badly the browser is affected... 

Logs show the stream starting successfully by Tunarr and then immediately killed. Streaming seems to work on other programs in the same system (VLC).

**Comment by dexxfm** [comment]
I have tested the web client on PC and it seems to work fine. I have an ipad, amazon tablet, and LG TV. i'll try all of these clients as well to see if it's just Plex for Windows.

**Comment by dexxfm** [comment]
Tested clients below:
iOS Plex App - Worked as designed
LG WebOS Plex App - Worked as designed
Amazon Fire Plex App - Worked as designed
Plex for Windows App - "The live session has ended" is the error message.

**Comment by twofacedsquare** [comment]
Same issue here

2024-10-22T12:46:45.511Z [info]: First stretch to 1440 x 1080. Then add padding to make it 1920 x 1080  {"channel":"07e8cd49-3fcc-4069-86a7-2442a73ea517"}
2024-10-22T12:46:53.514Z [info]: Concat Wrapper FFMPEG exited. (signal=SIGTERM, code=-1, expected?=true) {"args":"-nostdin -threads 1 -hide_banner -loglevel warning -user_agent Ffmpeg Tunarr/0.14.1 -nostats -fflags +genpts+discardcorrupt+igndts -reconnect 1 -reconnect_at_eof 1 -readrate 1 -i http://localhost:8000/stream/channels/07e8cd49-3fcc-4069-86a7-2442a73ea517.m3u8?mode=hls -map 0 -c copy -f mpegts pipe:1"}
2024-10-22T12:46:53.518Z [error]: Received error from session. Shutting down {"sessionType":"hls_concat","channelId":"07e8cd49-3fcc-4069-86a7-2442a73ea517"}
    error: {
      "code": null,
      "cmd": "C:\\Applications\\ffmpeg\\bin\\ffmpeg.exe -nostdin -threads 1 -hide_banner -loglevel warning -user_agent Ffmpeg Tunarr/0.14.1 -nostats -fflags +genpts+discardcorrupt+igndts -reconnect 1 -reconnect_at_eof 1 -readrate 1 -i http://localhost:8000/stream/channels/07e8cd49-3fcc-4069-86a7-2442a73ea517.m3u8?mode=hls -map 0 -c copy -f mpegts pipe:1"
    }
2024-10-22T12:46:53.521Z [error]: Error playing video {"code":null,"cmd":"C:\Applications\ffmpeg\bin\ffmpeg.exe -nostdin -threads 1 -hide_banner -loglevel warning -user_agent Ffmpeg Tunarr/0.14.1 -nostats -fflags +genpts+discardcorrupt+igndts -reconnect 1 -reconnect_at_eof 1 -readrate 1 -i http://localhost:8000/stream/channels/07e8cd49-3fcc-4069-86a7-2442a73ea517.m3u8?mode=hls -map 0 -c copy -f mpegts pipe:1"}


Works on Android TV + Phone no issues

Tried with ersatztv and had no issues

**Comment by chrisbenincasa** [comment]
I'm wondering if this next plex media server release will fix this: https://forums.plex.tv/t/plex-media-server/30447/659

Note this change: "(Livetv) Transcoding mpeg2 livetv streams would fail on windows (PM-2241)"

Were you using mpeg2 as the encoder here @dexxfm?

**Comment by chrisbenincasa** [comment]
Wondering if this could be resolved in a recent Plex update: 

![image](https://github.com/user-attachments/assets/c8c8961d-1c07-4f27-a51d-764479fb54ff)

This sounds exactly what could be causing this issue

**Comment by dexxfm** [comment]
It still doesn't work for me. On version 1.41.4.9463

**Comment by chrisbenincasa** [comment]
@dexxfm and the latest version of Tunarr?

I should be able to test this correctly now that I have a Windows workstation setup.

**Comment by nstjuliana** [comment]
@chrisbenincasa Client-Side Workaround https://www.reddit.com/r/PleX/comments/17jzbrn/plex_live_tv_this_live_tv_session_has_ended_andor/

In Plex for Windows App:
1. Settings > Debug > Disable Direct Stream
2. Settings > Languages > Subtitle Mode > Manually Selected or Shown with foreign audio

**Comment by jjjonesjr33** [comment]
> [@chrisbenincasa](https://github.com/chrisbenincasa) Client-Side Workaround https://www.reddit.com/r/PleX/comments/17jzbrn/plex_live_tv_this_live_tv_session_has_ended_andor/
> 
> In Plex for Windows App:
> 
>     1. Settings > Debug > Disable Direct Stream
> 
>     2. Settings > Languages > Subtitle Mode > Manually Selected or Shown with foreign audio

This worked for me.

**Comment by Aesculapius1** [comment]
> [@chrisbenincasa](https://github.com/chrisbenincasa) Client-Side Workaround https://www.reddit.com/r/PleX/comments/17jzbrn/plex_live_tv_this_live_tv_session_has_ended_andor/
> 
> In Plex for Windows App:
> 
>     1. Settings > Debug > Disable Direct Stream
> 
>     2. Settings > Languages > Subtitle Mode > Manually Selected or Shown with foreign audio

Worked for me as well.

**Comment by GigaGrim** [comment]
This appears to also affect the Plex Desktop app on Linux.

Flatpak version 1.109.0 tested on Fedora 42
Tested on 2 clients, all other Plex apps work as expected. (Android, Android TV, Web, etc.)

The above fix also worked on Linux, but only Disabling Direct Stream was required.

**Comment by queenkjuul** [comment]
I am using Plex HTPC for linux via the official snap (1.70.1) on Ubuntu 24.10, and the issue seems to exist there as well. Disabling Direct Stream does also seem to fix it, though it's in Settings > Video there (no Debug menu on the HTPC client)
